include/ofi_abi.h

```diff
@@ -111,7 +111,7 @@ extern "C" {
  * name appended with the ABI version that it is compatible with.
  */
 
-#define CURRENT_ABI "FABRIC_1.7"
+#define CURRENT_ABI "FABRIC_1.8"
```

Comment:
@j-xiong 

This change indicates that the fi_fabric2() call is not aligned with the documentation in this file.  Checking the code, it's using DEFAULT_SYMVER(... FABRIC_1.8) directly instead of CURRENT_SYMVER().  There were also changes to fi_getinfo / fi_freeinfo / fi_dupinfo as part of ABI 1.8 changes.  The changes to those functions should be captured in the man page below as well.

If CURRENT_SYMVER() will no longer be used, that macro should be removed and the man page updated to reflect direct use of DEFAULT_SYMVER instead.

Reply:
@shefty @j-xiong If we are going to implement changes to `fi_info` today. Is it better make it part of FABRIC_1.8 (since 2.0 is not released yet?), or we need to bump it as FABRIC_1.9?

Reply:
We will keep it 1.8 for the 2.0 release.

Reply:
@shefty Using DEFAULT_SYMVER() here for the new function instead of CURRENT_SYMVER() has the advantage of showing the exact ABI version at the definition and not needing to change it again when the ABI version is bumped. I think we can remove CURRENT_SYMVER() definition and usage info from the header. If no objection, I will make that change in a separate PR.

Reply:
I think it's fine to remove CURRENT_SYMVER